db/schema.rb

```diff
@@ -135,17 +135,14 @@
   end
 
   create_table "data_source_downloads", force: :cascade do |t|
-    t.string   "url",             limit: 255
-    t.text     "post_parameters", limit: 65535
-    t.string   "save_path",       limit: 255
-    t.datetime "created_at"
-    t.datetime "updated_at"
-    t.string   "handle",          limit: 255
-    t.string   "file_to_extract", limit: 255
-    t.string   "sheet_override",  limit: 255
-    t.text     "notes",           limit: 65535
+    t.integer  "data_source_id",         limit: 4
+    t.integer  "download_id",            limit: 4
+    t.datetime "last_file_vers_used",                default: '1970-01-01 00:00:00', null: false
+    t.string   "last_eval_options_used", limit: 255
```

Comment:
eval is longer than 255, depending on how you're using this column, if there is a change bast the 255th char, we wouldn't get new data. Unless you're using a hash of the eval options.